frontend/index_es.html

```diff
@@ -0,0 +1,551 @@
+
+		<!-- == GLOBAL APPLICATION SCRIPTS == -->
+		<!-- Bootstrap -->
+		<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
+
+		<!-- Global scripts -->
+		<script src="client/js/app.min.js"></script>
+
+		<!-- Site application scripts -->
+		<script src="client/js/plugins.min.js"></script>
+		<script src="client/js/app.min.js"></script>
+
+		<!-- Custom scripts -->
+		<script type="text/javascript">
+      /*
+      * Copied and adapted from the phone `dynamic_config.js`
+      * Adapted by supporting program name via parameter to make it
+      * easier to test against localhost
+      */
+      const _getStudyName = function(connectUrl) {
+        const _getFromParamOrDefault = function() {
+          const param_study_name = new URL(connectUrl).searchParams.get("study_config");
+          return param_study_name == null? "stage-program" : param_study_name;
+        }
+        const orig_host = new URL(connectUrl).hostname;
+        // If we are running this on dev, use a param instead to enable testing
+        if (orig_host == "localhost") { return _getFromParamOrDefault(); }
+        const first_domain = orig_host.split(".")[0];
+        if (first_domain == "openpath-stage") { return _getFromParamOrDefault(); }
+        const openpath_index = first_domain.search("-openpath");
+        if (openpath_index == -1) { return undefined; }
+        const study_name = first_domain.substr(0,openpath_index);
+        return study_name;
+      }
+			$(function() {
+				$(document).ready(function() {
+					const STUDY_NAME = _getStudyName(window.location);
+          const JOIN_CODE = `nrelopenpath://join_study?label=${STUDY_NAME}&source=github`
+
+          // Generate study-join QR code
+          var qrcode = new QRCode(document.getElementById("qrcode"), {
+	          text: JOIN_CODE,
+	          width: 100,
+	          height: 100,
+	          colorDark : "#000000",
+	          colorLight : "#ffffff",
+	          correctLevel : QRCode.CorrectLevel.H
+          });
+          $.getJSON(`https://raw.githubusercontent.com/e-mission/nrel-openpath-deploy-configs/main/configs/${STUDY_NAME}.nrel-op.json`, function(data) {
+            
+            // Title card
+            const PAGE_TITLE_TEXT = `${data.intro.translated_text.es.deployment_name} Pagina Inicial` 
+            $('#pageTitle').text(PAGE_TITLE_TEXT)
+            document.title = PAGE_TITLE_TEXT;
+            
+            // Navigation links
+            const PUBLIC_DASHBOARD_LINK = data.server.connectUrl.replace("api", "public");;
+            $('#publicDashboardLink').attr('href', PUBLIC_DASHBOARD_LINK);
+
+            // Deployment partner content text
+            const DEPLOYMENT_PARTNER_NAME = `${data.intro.deployment_partner_name}`;
+            $('#deploymentPartnerName').text(DEPLOYMENT_PARTNER_NAME);
```

Comment:
Per my final comments in 
https://github.com/e-mission/nrel-openpath-join-page/pull/2#pullrequestreview-1229686843

```suggestion
            const DEPLOYMENT_PARTNER_NAME = `${data.intro.translated_text.es.deployment_partner_name}`;
            $('#deploymentPartnerName').text(DEPLOYMENT_PARTNER_NAME);
```

Reply:
It will be done.